charts/argocd-notifications/templates/secret.yaml

```diff
@@ -27,4 +27,13 @@ stringData:
         {{- $v | toYaml | nindent 8 }}
 {{- end }}
 {{- end }}
+{{- if .Values.secret.notifiers.email.enabled }}
+    email:
+      host: {{ .Values.secret.notifiers.email.host }}
+      port: {{ .Values.secret.notifiers.email.port }}
+      insecure_skip_verify: {{ .Values.secret.notifiers.email.insecure_skip_verify }}
+      username: {{ .Values.secret.notifiers.email.username }}
+      password: {{ .Values.secret.notifiers.email.password }}
+      from: {{ .Values.secret.notifiers.email.from }}
```

Comment:
Should any of these `| quote`?

Reply:
In my tests all was working even without quote but I completely agree with you: it's safer to quote strings. Thank you for pointing that out! I did the change and all is working as expected. I pushed the commit, let me know if there's anything else I can do :-)